bip-0442.md

```diff
@@ -90,138 +91,187 @@ highly optimized for just 2 stack elements.
 <a> <b> <c> | PC PC <pc-hash> OP_EQUALVERIFY
 ```
 
-### Use in LN-Symmetry
-
-To do LN-Symmetry contracts that don't require the nodes to keep old states,
-we need to solve the data availability problem presented by unilateral closes.
-Channel peers must be able to reconstruct the script that spends an
-intermediate state.
-
-Using in sequence `OP_CHECKTEMPLATEVERIFY`, `OP_PAIRCOMMIT`, `OP_INTERNALKEY`
-and `OP_CHECKSIGFROMSTACK` we can construct a rebindable channel that is also
-[optimal].
-
-The following assembly-like pseudo-code shows a possible LN-Symmetry channel
-construction that provides data availability to spend to the latest state from
-an earlier state pushed on-chain with a forced close by channel partner.
-
+### Use in Lightning Symmetry
```

Comment:
I think "LN-Symmetry" is more commonly used to refer to the concept than "Lightning Symmetry". Is there a specific reason why you prefer the latter?